tools/reactive-controllers/src/SpectrumDelegates.ts

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import type { ReactiveController } from 'lit';
+import { SpectrumElement } from '@spectrum-web-components/base';
+import { DownStateController } from './delegate-controllers/DownStateController.js';
+
+export class SpectrumDelegates implements ReactiveController {
+    private host: SpectrumElement;
+
+    private downStateController: DownStateController | null = null;
+
+    set theme(value: string | null) {
+        if (value === 'spectrum-two') {
+            this.downStateController = new DownStateController(this.host);
+        } else {
+            this.downStateController?.unmanage();
+        }
```

Comment:
This isn't the logic by which we chose to add this, right? It should be something like:
```
const { spectrumConfig } = this.host.constructor;
Object.entries(spectrumConfig).forEach(([controller, themes]) => {
    if (themes.includes(value)) {
        this[controller] = new Delegates[Controller](this.host);
    }
});
```
This way both multiple controllers can be added and individual controllers can be added on multiple themes?

Separately, should `theme` now be `system`?